source/Calamari.Tests/KubernetesFixtures/ManifestReporterTests.cs

```diff
@@ -40,16 +41,21 @@ public void GivenValidYaml_ShouldPostSingleServiceMessage(string enabledFeatureT
             var memoryLog = new InMemoryLog();
             var variables = new CalamariVariables();
             variables.Set(KnownVariables.EnabledFeatureToggles, enabledFeatureToggle);
-
-            var yaml = @"foo: bar";
-            var expectedJson = "{\"foo\": \"bar\"}";
+            
+            //Test that quotes are preserved, especially for numbers
+            var yaml = @"name: George Washington
+alphafield: ""fgdsfsd""
+age: 89
```

Comment:
nit: George Washington is either 67 or 292 - depending on whether you consider death to stop aging :stuck_out_tongue: